JSDemos/Demos/Charts/DoughnutWithCustomLabelInCenter/jQuery/index.js

```diff
@@ -29,18 +29,69 @@ $(() => {
         .getVisiblePoints()
         .reduce((s, p) => s + p.originalValue, 0);
       const { country } = pieChart.getAllSeries()[0].getVisiblePoints()[0].data;
-      const content = $(`<svg><circle cx="100" cy="100" fill="#eee" r="${pieChart.getInnerRadius() - 6}"></circle>`
-                    + `<image x="70" y="58" width="60" height="40" href="../../../../images/flags/${country.replace(/\s/, '').toLowerCase()}.svg"/>`
-                    + '<text text-anchor="middle" style="font-size: 18px" x="100" y="120" fill="#494949">'
-                    + `<tspan x="100" >${country}</tspan>`
-                    + `<tspan x="100" dy="20px" style="font-weight: 600">${
-                      formatNumber(total)
-                    }</tspan></text></svg>`);
-
-      container.appendChild(content.get(0));
+
+      const circle = createCircle(100, 100, '#eee', pieChart.getInnerRadius() - 6);
+      const image = createImage(70, 58, 60, 40, `../../../../images/flags/${country.replace(/\s/, '').toLowerCase()}.svg`);
```

Comment:
It's hard to understand what these parameters mean seeing on this call. Maybe it's better to use an [object parameter](https://refactoring.guru/introduce-parameter-object) here?
We cab also reduce a copypaste if use an object parameter and set attributes in a circle

Like
```javascript
      const image = createElement('image', {
          x: 70, 
          y: 58, 
          width: 60, 
          height: 40, 
          href: `../../../../images/flags/${country.replace(/\s/, '').toLowerCase()}.svg`
      });
      
      function createElement(type, attributes) {
       const element = document.createElementNS('http://www.w3.org/2000/svg', type);
       for(attributeName in attributes) {
            element.setAttribute(attributeName, attributes[attributeName])  
       }
        return element;
      }
```